app/src/shared/components/ImageUpload/index.jsx

```diff
@@ -53,9 +53,10 @@ const ImageUpload = ({
             setUploaded(false)
 
             if (setImageToUpload) {
-                setImageToUpload(((Object.assign({}, image, {
+                // $FlowFixMe property `preview` is missing in  `File`.
```

Comment:
Had to revert this because for some reason Flow casting removes other properties from the result besides `path`and `preview`. Ping @mondoreale